segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -1040,24 +1039,22 @@ public void readTableEntriesDelta(WireCommands.ReadTableEntriesDelta readTableEn
 
     }
 
-    private void truncateIndexSegment(String segment, long offset) {
+    private CompletableFuture<Void> truncateIndexSegment(String segment, long offset) {
```

Comment:
This is changed to returning a completableFuture. But the return value isn't used in the chaining. It should be made into a thenCompose.

Reply:
Made the changes.